### ISSUE TITLE
Why don't have yet a release in PyPI?

### ISSUE THREAD

**Issue by ryukinix** [renpy/pygame_sdl2#39]
I think woud be cool if the project was deployed to PyPI.

**Comment by TTimo** [comment]
+1 Make it happen guys!

I will test this out with a reasonably complex Windows/Linux/OSX game once I can easily grab it from PyPI.

**Comment by metulburr** [comment]
Why would you require in pypi to test it out?

**Comment by TTimo** [comment]
Not required, just convenience. I have enough work as it is, I don't want to try pygame_sdl2 badly enough to pull the git repo and do more work setting it up. Once it lands on PyPI, it's an indication that it's getting closer to broader consumption for end users.

**Comment by sam0s** [comment]
I can't even figure out how to install this on windows, and I'd really like to try it. This is the error I am getting for anyone who can help: 
`C:\Users\Sam Tubb\Desktop\pygame_sdl2-master>python setup.py install
pygame_sdl2.error is out of date.
Traceback (most recent call last):
  File "setup.py", line 109, in <module>
    cython("pygame_sdl2.error", libs=sdl_libs)
  File "C:\Users\Sam Tubb\Desktop\pygame_sdl2-master\setuplib.py", line 245, in cython
    c_fn])
  File "C:\Python27\lib\subprocess.py", line 499, in check_call
    retcode = call(*popenargs, **kwargs)
  File "C:\Python27\lib\subprocess.py", line 486, in call
    return Popen(*popenargs, **kwargs).wait()
  File "C:\Python27\lib\subprocess.py", line 672, in __init__
    errread, errwrite)
  File "C:\Python27\lib\subprocess.py", line 882, in _execute_child
    startupinfo)
WindowsError: [Error 2] The system cannot find the file specified`

Would be nice to have this on PyPi

**Comment by Connor124** [comment]
Try installing Cython. I had an error very similar to this.

**Comment by sam0s** [comment]
@Connor124 will do thanks!

**Comment by erikbgithub** [comment]
Ping. This is a valid question. It's also not that hard. Does this need some compiled code or something?

**Comment by olymk2** [comment]
Any progress on this ?
